core/src/test/java/org/infinispan/lock/LockContainerHashingTest.java

```diff
@@ -11,25 +11,26 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.concurrent.locks.Lock;
 
 @Test(groups = "unit", testName = "lock.LockContainerHashingTest")
 public class LockContainerHashingTest extends AbstractInfinispanTest {
-   private AbstractStripedLockContainer<?> stripedLock;
+   private StripedLockContainer stripedLock;
```

Comment:
Since you've extracted the hashing functionality in the `StripedHashFunction` class, I think you could test it here directly.